### ISSUE TITLE
loss : nan after training second batch size with convolution 1D

### ISSUE THREAD

**Issue by darzok** [fchollet/keras#949]
hello, I'm trying to code something like that http://arxiv.org/abs/1506.04214
to start i'm trying to implement a simple conv 1D on time series (forex). 

the goal it's to predict the 10th value of the last value of a temporal window (60 values in window and I want to predict the 70th) To have my data compatible with conv 1D I prepare my data into a array of 5019 matrix of 60 raw and 1 column

my code : 

```javascript
sizeTraining=0.9
sizeTest=0.1
window=60
h=10

print('Loading data ...')
forex = FOREX()
(X_train, y_train),(X_test,y_test) = forex.load_data(sizeTraining, sizeTest, window, h)

Xtrain_mean = X_train.mean()
Xtrain_ecart_type = X_train.std()
X_train=(X_train-Xtrain_mean)/Xtrain_ecart_type

Xtest_mean = X_test.mean()
Xtest_ecart_type = X_test.std()
X_test=(X_test-Xtest_mean)/Xtest_ecart_type

ytrain_mean = y_train.mean()
ytrain_ecart_type = y_train.std()
y_train=(y_train-ytrain_mean)/ytrain_ecart_type

ytest_mean = y_test.mean()
ytest_ecart_type = y_test.std()
y_test=(y_test-ytest_mean)/ytest_ecart_type

print('x reshape')
X_train = np.reshape(X_train, (-1,window,1))
X_test = np.reshape(X_test, (-1,window,1))
print(X_train.shape)
print(X_train)

print('y reshapet')
y_train = np.reshape(y_train, (-1,1))
y_test = np.reshape(y_train, (-1,1))
print(y_train.shape)

model = Sequential()
#premiere couche de convolution
model.add(Convolution1D(nb_filter=60,filter_length=3,border_mode="full",activation="relu",subsample_length=1,input_dim=1,input_length=60))
conv1= Activation('relu')
model.add(conv1)
#premiere couche de subsampling
model.add(MaxPooling1D(pool_length=2))
model.add(Dropout(0.25))

model.add(Convolution1D(nb_filter=60,filter_length=3,border_mode="full",activation="relu",subsample_length=1))
conv1= Activation('relu')
model.add(conv1)
#premiere couche de subsampling
model.add(MaxPooling1D(pool_length=2))
model.add(Dropout(0.25))

#on applatit la sortie pour la presenter au MLP
model.add(Flatten())

#premiere couche du MLP avec activation de type maxoutdense
model.add(Dense(output_dim=500))
model.add(Dropout(0.5))

model.add(Dense(output_dim=500))
model.add(Dropout(0.5))

model.add(Dense(1))
model.add(Activation('linear'))

model.compile(loss='mse', optimizer='adadelta')

history = model.fit(X_train, y_train, batch_size=128, nb_epoch=2, show_accuracy=False, verbose=1, validation_data=(X_test, y_test))
score = model.evaluate(X_test, y_test, show_accuracy=False, verbose=0)
'''

this my problem : 

```javascript
Epoch 1/2

 128/5019 [..............................] - ETA: 24s - loss: 0.8462
 256/5019 [>.............................] - ETA: 24s - loss: nan   
 384/5019 [=>............................] - ETA: 23s - loss: nan
 512/5019 [==>...........................] - ETA: 22s - loss: nan
 640/5019 [==>...........................] - ETA: 21s - loss: nan
'''

I think my input data is correct so I don't know why I have this ..

**Comment by mmmikael** [comment]
Note that you adding ```ReLU``` twice after each convolution. If you specify it in the arguments of ```Convolution1D```, no need to add another layer.

**Comment by darzok** [comment]
aah ! yes ! my mistake ^^ I'm going to try without it and hope it's gonna be better :) thank you very much
